Glyssen/Dialogs/VoiceActorAssignmentDlg.cs

```diff
@@ -39,6 +42,28 @@ public VoiceActorAssignmentDlg(Project project)
 
 				foreach (KeyValuePair<int, CharacterGroup> pair in charGroupTemplate.CharacterGroups)
 					characterGroups.Add(pair.Value);
+
+				Dictionary<string, int> keyStrokesByCharacterId = new Dictionary<string, int>();
+				foreach (var book in m_project.IncludedBooks)
+				{
+					foreach (var block in book.GetScriptBlocks(true))
+					{
+						if(!keyStrokesByCharacterId.ContainsKey(block.CharacterId))
+							keyStrokesByCharacterId.Add(block.CharacterId, 0);
+						keyStrokesByCharacterId[block.CharacterId] += block.GetText(false).Length;
+					}
+				}
+
+				foreach (var group in characterGroups)
+				{
+					int keyStrokes = 0;
+					foreach (var characterId in group.CharacterIds)
+					{
+						if (keyStrokesByCharacterId.ContainsKey(characterId))
+							keyStrokes += keyStrokesByCharacterId[characterId];
+					}
+					group.EstimatedHours = (keyStrokes/kKeyStrokesPerHour).ToString("0.00");
```

Comment:
Let's see if EstimatedHours can be a double to ensure proper sorting in the grid